lib/PACEdit.pm

```diff
@@ -225,6 +225,33 @@ sub __checkRBAuth {
             _($self, 'alignExpect')->set_has_tooltip(0);
 
         }
+        _($self, 'rbUseProxyJump')->set_label("Use Jump Server");
+        my $status = _($self, 'rbUseProxyJump')->get_active();
+        _($self, 'rbUseProxyJump')->set_sensitive(1);
+        _($self, 'rbUseProxyJump')->set_tooltip_text("An alternative to SSH tunneling to access internal machines through gateway");
+        _($self, 'vboxJumpCfgOptions')->set_sensitive(_($self, 'rbUseProxyJump')->get_active());
+        _($self, 'vboxJumpCfgOptions')->set_sensitive($status);
+        _($self, 'vboxJumpCfg')->set_visible(1);
+        _($self, 'vboxCfgManualProxyConn')->set_visible(1);
+    } elsif (_($self, 'comboMethod')->get_active_text() =~ /RDP|VNC/) {
+        _($self, 'rbUseProxyJump')->set_sensitive(1);
+        _($self, 'vboxJumpCfgOptions')->set_sensitive(1);
+        _($self, 'rbUseProxyJump')->set_label("SSH Tunnel port forward");
+        _($self, 'rbUseProxyJump')->set_tooltip_text("Open SSH tunnel for this connection");
+        _($self, 'vboxJumpCfgOptions')->set_sensitive(_($self, 'rbUseProxyJump')->get_active());
+        _($self, 'vboxJumpCfg')->set_visible(1);
+        _($self, 'vboxCfgManualProxyConn')->set_visible(0);
+    } else {
+        if (_($self, 'rbUseProxyJump')->get_active()) {
+            _($self, 'rbUseProxyIfCFG')->set_active(1);
+        }
+        _($self, 'vboxJumpCfg')->set_visible(0);
+        _($self, 'rbUseProxyJump')->set_sensitive(0);
+        _($self, 'vboxJumpCfgOptions')->set_sensitive(0);
+        _($self, 'rbUseProxyJump')->set_label("SSH Tunnel port forward");
```

Comment:
To have the same wording ; shouldn't we label this "Use SSH tunnel" ?

Reply:
Done